app/src/main/res/values-night/color_palette.xml

```diff
@@ -90,4 +90,6 @@
   <color name="color_palette_confetti_red_color">@color/color_def_confetti_rosy_pink</color>
   <color name="color_palette_confetti_yellow_color">@color/color_def_confetti_bright_gold</color>
   <color name="color_palette_confetti_blue_color">@color/color_def_confetti_crystal_blue</color>
+  <color name="color_palette_forward_arrow_button_color">@color/color_def_oppia_turquoise</color>
```

Comment:
This change should be removed once your part-2 is merged

Reply:
> This change should be removed once your part-2 is merged

okay sure

Reply:
> This change should be removed once your part-2 is merged

Now Part 2 is merge, I take pull from the develop branch but if I remove this value then dark mode this component is not work. As after pull this not add new value that's why?

https://github.com/oppia/oppia-android/blob/f99fa1d8653e23f8d22e1584a9fb1793aaa19acd/app/src/main/res/values/component_colors.xml#L35

This component using that values.

Reply:
Just make sure that this value is getting used somwhere.

Earlier when I reviewed it. the code was written in such a way that it looked as if it was not getting used anywhere.

If it is getting used anywhere, don't remove it.

Reply:
> Just make sure that this value is getting used somwhere.
> 
> Earlier when I reviewed it. the code was written in such a way that it looked as if it was not getting used anywhere.
> 
> If it is getting used anywhere, don't remove it.

Okay I compare it develop and it from develop and it's used in layout.